src/llmcompressor/modifiers/transform/awq/mappings.py

```
@@ -225,12 +250,12 @@ class AWQMapping:
     "Gemma2ForCausalLM": _gemma_mappings,
     "Gemma3ForCausalLM": _gemma_mappings,
     "Gemma3ForConditionalGeneration": _gemma_mappings,
-    "Glm4MoeForCausalLM": default_mappings,
+    "Glm4MoeForCausalLM": _moe_default_mappings,
     "GlmMoeDsaForCausalLM": _deepseek_mappings,
-    "LlamaForCausalLM": default_mappings,
-    "Llama4ForConditionalGeneration": default_mappings,
-    "Mistral3ForConditionalGeneration": default_mappings,
-    "MistralForCausalLM": default_mappings,
+    "LlamaForCausalLM": _default_mappings,
```

Ruff (0.15.11)

[error] Undefined name `_default_mappings`

(F821)